### ISSUE TITLE
Anouncing n3n: A fork with improvements and no CLA required

### ISSUE THREAD

**Issue by hamishcoleman** [ntop/n2n#1171]
I would like to announce the first stable release of [n3n](https://github.com/n42n/n3n/).

This is a fork with different licensing and the aim to remain protocol compatible with n2n.  

Since neither of the maintainers of this n2n project have been happy to contribute under the new ntop Contributor License Agreement (CLA), it is unfortunately time to create a fork. See #1156 for some discussion about the license.  

It is also important for n2n users to realise that the n2n project appears to be unmaintained - the ntop team have not made any ticket triage or code changes in over six months.

Major improvements since the fork from n2n:

- Documentation
- Command line processing simplification
- Configuration file with clear descriptions and help texts
- JsonRPC API with built-in web server
- Significantly improved Debian and Redhat packaging
- More automated tests and builds

**Comment by lucaderi** [comment]
@hamishcoleman Said that you are free to do what you like, the CLA protects contributors and developers that the code that is incorporated in n2n is original, is not copied violating the original coder's license, and that it is patent-free so users should not be concerned when using it. If making a fork for circumventing all this means more freedom to you, I have a different opinion.

**Comment by hamishcoleman** [comment]
@lucaderi, I would be happy to discuss this in #1156

**Comment by fengdaolong** [comment]
I agree with you about keeping n2n alive rather than slowly dying, but n3n is a weird name.

n2n is a very good name. Its meaning can be net to net or nat to nat. The branch can be renamed n2n-ng or n2n+?
The following name suggestions from ChatGPT:

n2m：net to mesh
n2c：net to cloud
n2p：net to peer
n2s：net to stream
n2h：net to hub
n2f：net to frontier
n2v：net to vault
n2x：net to expand
n2d：net to direct
n2r：net to reach
n2g：net to gateway
n2q：net to quick
n2o：net to open
n2y：net to yield
n2l：net to link
n2u：net to unite
n2i：net to interface
